pypeit/edgetrace.py

```diff
@@ -268,10 +268,6 @@ class EdgeTraceSet(DataContainer):
         qa_path (:obj:`str`, optional):
             Directory for QA output. If None, no QA plots are
             provided.
-        bpm (`numpy.ndarray`_, optional):
```

Comment:
I'm not following why bpm is being removed as an input. I know for certain that this was being used to construct the Sobel image as I recall fixing a bug in how that was constructed, and that involved the bpm?